mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -305,6 +305,11 @@ impl NexusChild {
         &self.name
     }
 
+    /// Get name of the nexus this child belongs to.
+    pub fn get_nexus_name(&self) -> &str {
```

Comment:
This might be more appropriate as  `parent()`?

Reply:
Parent here is nexus. :-) There are no other parents for nexus children than a nexus.
We will probably change 'parent' member to 'nexus' or so.